scripts/west_commands/boards.py

```diff
@@ -0,0 +1,104 @@
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import collections
+import os
+import re
+import shutil
+import subprocess
+import sys
+import textwrap
+
+from west import log
+from west import cmake
+from west.configuration import config
+from west.commands import WestCommand
+from cmake import run_cmake
+
+BOARDS_DESCRIPTION = '''\
```

Comment:
No need for a multiline triple quoted string here; this is a short string which will fit just fine as an argument to super().

Reply:
I did this for consistency with other commands, but good point, no need.

Reply:
The other commands should only be using triple quotes for multiline strings too; if not, I'll fix.